charts/kruise/v0.8.1/values.yaml

```diff
@@ -2,6 +2,7 @@
 
 crds:
   managed: true
+  namespace: kruise-system
```

Comment:
`namespace` is inappropriate to be defined in `crds`, because CRDs have no namespace. It is for the other resources like pods, service, configmap.

Maybe a new `installation`:

```yaml
installation:
  namespace: kruise-system
```